### PR TITLE
P0 button

### PR DESCRIPTION
**Pull request by vidhi-thakur**

### CODE REVIEW

File: components/component_style/badge.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    bottom: 10px;
+    left: 10px;
+    cursor: pointer;
+}
+
+.icon-badge {
+    background-color: white;
```
Comment (nyyshaaa) [inline components/component_style/badge.css:64]: you can also put white in var, to maintain consistency.Although i have done similar mistakes but try to remember from next time.

**Review by nyyshaaa — COMMENTED** [review]
looks good overall, just that color yellow looks somewhat light, you can give it somewhat different shade.

File: components/component_style/badge.css (hunk 1 above)
Comment (vidhi-thakur) [reply components/component_style/badge.css:64]: will do, thanks for suggestion

**Comment by vidhi-thakur** [comment]
> looks good overall, just that color yellow looks somewhat light, you can give it somewhat different shade.

i do plan on changing primary color in future, thanks for you review. also could you mention you discord name. I can't find you on discord

**Pull request merged**